workspaces/src/types.rs

```diff
@@ -5,6 +5,7 @@ use std::path::Path;
 
 pub use near_account_id::AccountId;
 pub(crate) use near_crypto::{KeyType, Signer};
+pub use near_primitives::types::BlockId;
```

Comment:
Perhaps we should have a local type to not be affected by the updates to near_primitives as well as making ours more ergonomic/better docs and just swap the type internally?

Reply:
Let's go with the version you mentioned later about abstracting it